.travis.yml

```diff
@@ -22,13 +22,13 @@ matrix:
       script:
         - mkdir build && pushd build
         - cmake -DCMAKE_INSTALL_PREFIX=~/.local -DWITH_CXX=ON ..
-        - make install -j2 && make tests -j2 && ctest -j2 --output-on-failure
+        - make install -j2 && ctest -j2 --output-on-failure
```

Comment:
I'm confused, I thought that the point of this patch is that the tests for `ctest` are already built?  Why is `make tests` needed?

(Note also that you can just do `make -j2 install tests` to build multiple targets; this is better because that way it has more potential parallelism.)